v0/destinations/fb/transform.js

```diff
@@ -253,8 +248,11 @@ function buildBaseEvent(message) {
   } else if (isAppleFamily(sourceSDK)) {
     sourceSDK = "i2";
   } else {
-    // if the sourceSDK is not android or ios, send an empty string
-    sourceSDK = "";
+    // if the sourceSDK is not android or ios
+    throw new CustomError(
+      'Extended Device Information i.e, "context.device.type" is required',
```

Comment:
Just directly commit this suggestion,
```suggestion
      'Extended device information i.e, "context.device.type" is required',
```

Reply:
Done